packages/block-editor/src/components/link-control/use-internal-input-value.js

```diff
@@ -13,8 +13,8 @@ export default function useInternalInputValue( value ) {
 		 * If the value's `text` property changes then sync this
 		 * back up with state.
 		 */
-		if ( value?.title && value.title !== internalInputValue ) {
-			setInternalInputValue( value.title );
+		if ( value && value !== internalInputValue ) {
+			setInternalInputValue( value );
```

Comment:
Ignore - will be rebased out by https://github.com/WordPress/gutenberg/pull/46113